testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -1114,6 +1114,39 @@ QUnit.module('Lookup', {
         assert.equal(this.popup.option('height'), popupHeight, 'popupHeight applied to popover');
     });
 
+    [true, false].forEach((usePopover) => {
+        QUnit.test(`overlay should contain role="dialog" when usePopover: ${usePopover}`, function(assert) {
+            const instance = $('#lookup').dxLookup({
+                usePopover: usePopover
+            }).dxLookup('instance');
+
+            this.togglePopup();
+
+            const $overlayContent = instance._popup.$overlayContent();
+
+            assert.equal($overlayContent.attr('role'), 'dialog');
```

Comment:
It's better always use `strictEqual`. U don't use `==` instead of `===`, right?
```suggestion
            assert.strictEqual($overlayContent.attr('role'), 'dialog');
```